serverless.yml

```diff
@@ -1,6 +1,5 @@
 service: hillbrook-traffic
-app: hillbrook-traffic
-org: kielni
+useDotenv: true
```

Comment:
1. Without `useDotenv: true`, I get the following error when I run `serverless invoke -f collect-sheet --log`:

```
Error:
Cannot resolve serverless.yml: Variables resolution errored with:
  - Cannot resolve variable at "provider.role": Value not found at "env" source,
  - Cannot resolve variable at "provider.iamRoleStatements.0.Resource": Value not found at "env" source,
  - Cannot resolve variable at "provider.iamRoleStatements.1.Resource": Value not found at "env" source,
  - Cannot resolve variable at "provider.iamRoleStatements.2.Resource": Value not found at "env" source,
  - Cannot resolve variable at "provider.environment.STATIONS": Value not found at "env" source,
  - Cannot resolve variable at "provider.environment.ALERT_ARN": Value not found at "env" source,
  - Cannot resolve variable at "provider.environment.GOOGLE_SHEET_ID": Value not found at "env" source
```

2. I had to remove the `app` and `org` keys because I got serverless permissions errors. I assume this is referencing @kielni's org configuration